### ISSUE TITLE
Edited json file is not displayed in the app

### ISSUE THREAD

**Issue by securespider** [securespider/pe#17]
To reproduce: copy over past json file back into the data directory

Expected: data should sync with the application

Actual: Data is not displayed in the application

SS:
![image.png](https://raw.githubusercontent.com/securespider/pe/main/files/40761317-76a0-4bab-92f0-cc4ca810d882.png)

**Comment by nus-pe-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Reasoning is the same as the bug on deleting of JSON file

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Deleting Json file does not clear the address book
>
>To reproduce: Delete json file
>
>Expected: Deleting json file changes the delivery list
>
>Actual: Delivery list still stays 
>
>SS:
>
>![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2223S2/pe/master/files/18aae801-1062-427f-98f5-cdcb0887f0b2.png)
>
>
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2223S2/pe-interim#1269] [original labels: type.FunctionalityBug severity.Medium]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>This works as intended, as Bookopedia was not relaunched. As shown in the SS below, if tester was to follow the steps correctly by launching Bookopedia only AFTER the changes to JSON data file, it would work as intended.
>
>
>![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2223S2/pe/master/files/5860c148-b95a-4f4a-b57a-9b52a6324639.png)

# Items for the Tester to Verify

## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
